### PR TITLE
Add a small boilerplate README with important links and references

### PR DESCRIPTION
**Pull request by awvwgk**
- explain what this repository is for and what is included
- which projects are more suitable starting points to start looking into
- how to engage with the fortran-lang community via discourse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+This repository holds our central community resources, like our community wide
+[code of conduct](CODE_OF_CONDUCT.md) and other similar documents.
+
+
+## Contributing to fortran-lang
+
+If you plan to contribute to fortran-lang, have a look at the other repositories at
+[@fortran-lang](https://github.com/fortran-lang).
+For some pointers where to go next from here, have a look at the
```
Comment (LKedward) [inline README.md:11]:
```suggestion
For some pointers where to go next from here, have a look at:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- [standard library](https://github.com/fortran-lang/stdlib)
+- [package manager](https://github.com/fortran-lang/fpm)
+- [webpage sources](https://github.com/fortran-lang/fortran-lang.org)
+
+
+## Contributing to this resources
```
Comment (LKedward) [inline README.md:18]:
```suggestion
## Contributing to this repository
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+You want to shape help this community? You are more than welcome.
```
Comment (LKedward) [inline README.md:20]:
```suggestion
Want to help shape this community? You are more than welcome.
```

**Review by LKedward — APPROVED** [review]
Thanks @awvwgk. Left a few minor comments, but otherwise looks good to me :+1:

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Community resources for fortran-lang
+
+This repository holds our central community resources, like our community wide
+[code of conduct](CODE_OF_CONDUCT.md) and other similar documents.
```
Comment (milancurcic) [inline README.md:4]:
```suggestion
This repository holds our central community resources, like the community-wide
￼[Code of Conduct](CODE_OF_CONDUCT.md) and other documents.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Contributing to fortran-lang
+
+If you plan to contribute to fortran-lang, have a look at the other repositories at
+[@fortran-lang](https://github.com/fortran-lang).
+For some pointers where to go next from here, have a look at:
+
+- [standard library](https://github.com/fortran-lang/stdlib)
+- [package manager](https://github.com/fortran-lang/fpm)
+- [webpage sources](https://github.com/fortran-lang/fortran-lang.org)
```
Comment (milancurcic) [inline README.md:15]:
```suggestion
If you want to contribute to fortran-lang, look at other ￼[@fortran-lang](https://github.com/fortran-lang) repositories:

￼- [Standard library](https://github.com/fortran-lang/stdlib)
￼- [Package manager](https://github.com/fortran-lang/fpm)
￼- [Website sources](https://github.com/fortran-lang/fortran-lang.org)
```

**Review by milancurcic — APPROVED** [review]
Looks great, thank you for starting it!

**Comment by milancurcic** [comment]
I'll go ahead and merge, thanks again @awvwgk!

**Pull request merged**